functions/Get-DbaDbccProcCache.ps1

```diff
@@ -0,0 +1,98 @@
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: DBCC
+        Author: Patrick Flynn (@sqllensman)
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Get-DbaDbccProcCache
+
+    .EXAMPLE
+        PS C:\> Get-DbaDbccProcCache -SqlInstance Server1
+
+        Get results of DBCC PROCCACHE for Instance Server1
+
+    .EXAMPLE
+        PS C:\> 'Sql1','Sql2/sqlexpress' | Get-DbaDbccProcCache
+
+        Get results of DBCC PROCCACHE for Instances Sql1 and Sql2/sqlexpress
+
+    .EXAMPLE
+        PS C:\> $cred = Get-Credential sqladmin
+        PS C:\> Get-DbaDbccProcCache -SqlInstance Server1 -SqlCredential $cred
+
+        Connects using sqladmin credential and gets results of DBCC PROCCACHE for Instance Server1 using
+
+    #>
+    [CmdletBinding()]
+    param (
+        [parameter(Mandatory, ValueFromPipeline)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [switch]$EnableException
+    )
+    begin {
+
+        $stringBuilder = New-Object System.Text.StringBuilder
+        $null = $stringBuilder.Append("DBCC PROCCACHE WITH NO_INFOMSGS")
+    }
+    process {
+        $query = $StringBuilder.ToString()
+        foreach ($instance in $SqlInstance) {
+            Write-Message -Message "Attempting Connection to $instance" -Level Verbose
+            try {
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+            } catch {
+                Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+            }
+
+            try {
+                if ($Pscmdlet.ShouldProcess($server.Name, "Execute the command $query")) {
+                    Write-Message -Message "Query to run: $query" -Level Verbose
+                    $results = $server.Query($query)
+                }
+            } catch {
+                Stop-Function -Message "Failure" -ErrorRecord $_ -Target $server -Continue
+            }
+            If ($Pscmdlet.ShouldProcess("console", "Outputting object")) {
+                foreach ($row in $results) {
+                    [PSCustomObject]@{
+                        ComputerName       = $server.ComputerName
+                        InstanceName       = $server.ServiceName
+                        SqlInstance        = $server.DomainInstanceName
+                        NumProcBuffs       = $row[0]
```

Comment:
`Proc` is probably okay because we're used to hearing proc but please remove Num and expand out Buffer or actually, it'd probably be implied so it could be `Count, Used, Active, CacheSize, CacheUsed, CacheActive`.

Reply:
Change has been made